addons/vehicles/stringtable.xml

```diff
@@ -19,5 +19,32 @@
             <Japanese>武装オフロード (黒)</Japanese>
             <French>Tout-terrain armé (Noir)</French>
         </Key>
+        <Key ID="STR_TACS_Vehicles_Polaris_Black">
+            <English>Polaris DAGOR (Black)</English>
+        </Key>
+        <Key ID="STR_TACS_Vehicles_Polaris_Armed_Black">
+            <English>Polaris DAGOR Armed (Black)</English>
```

Comment:
ACE3 strings [specify the weapon](https://github.com/acemod/ACE3/blob/d684a8042e56810fc916d9d205f228cb7de713ed/addons/realisticnames/stringtable.xml#L3210).
We should follow that same as for [BWA3 optional](https://github.com/Theseus-Aegis/TheseusServices/blob/master/optionals/variants_bwa3/stringtable.xml#L14).

Reply:
Names corrected